components/LinkCustom/LinkCustom.tsx

```diff
@@ -3,7 +3,6 @@
 
 import Link from 'next/link';
 import React, { JSX } from 'react';
-import { cn } from '@/utils/utils';
 
 interface ILinkCustomProps {
```

Comment:
I mention this because Shashi had me do it with my Alert Notification props. Make a seperate file in same folder named LinkCustom.interface.ts and move these props into it. Then import it into this file to be used.